cpp/include/cudf/datetime.hpp

```diff
@@ -207,5 +207,89 @@ std::unique_ptr<cudf::column> is_leap_year(
   rmm::mr::device_memory_resource* mr = rmm::mr::get_current_device_resource());
 
 /** @} */  // end of group
+
+/**
+ * @brief  Round up to the nearest day
+ *
+ * @param[in] cudf::column_view of the input datetime values
+ *
+ * @returns cudf::column of the same datetime resolution as the input column
+ * @throw cudf::logic_error if input column datatype is not TIMESTAMP
```

Comment:
Bring throw before `param`.
Also, `return`, not `returns`.

```suggestion
 * @return cudf::column of the same datetime resolution as the input column
```

Reply:
Same comment as before, previously in the file `throw` was after params and `returns` was used instead of `return`. So should I still change it?

Reply:
Just change your new code :)